netclient/ncutils/netclientutils.go

```diff
@@ -592,3 +593,24 @@ func ModPort(node *models.Node) error {
 	}
 	return err
 }
+
+// CheckIfDaemonExists - checks if netclient daemon is up and running
+func CheckIfDaemonExists() bool {
+	daemonExists := false
+	pid, err := ReadPID()
```

Comment:
pid will not work for windows, for windows you will need to check for the presence of winsw.exe

Reply:
@0xdcarns found this one pkg (https://github.com/mitchellh/go-ps) to check for running process looks like it works for windows as well

Reply:
actually, this check wouldn't be needed anymore will remove it since the option is changed back to `install`